pkg/core/serving/knative/servingrun.go

```diff
@@ -256,26 +255,22 @@ func (r *servingRun) createService(s *openfunction.Serving, cm map[string]string
 				case "min-scale", "minScale":
 					minScale = v
 				}
-				if _, exist := s.Spec.Annotations[fmt.Sprintf("%s/%s", knativeAutoscalingPrefix, k)]; !exist {
+				if _, exist := s.Spec.Annotations[fmt.Sprintf("autoscaling.knative.dev/%s", k)]; !exist {
 					s.Spec.Annotations[k] = v
```

Comment:
I think we can use the same key as the original, because below code is actually not working because it's lack of `autoscaling.knative.dev/` prefix
```go
s.Spec.Annotations[k] = v
```